packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -67,7 +70,9 @@ export const InboundLineEdit: FC<InboundLineEditProps> = ({
     }
   });
 
-  return (
+  return loading ? (
+    <BasicSpinner />
```

Comment:
would be good to add a text prop to the spinner so that this instance could say 'saving' rather than 'loading' - which could still be the default if no prop given